client/src/components/Card/Card.jsx

```diff
@@ -0,0 +1,20 @@
+import React, { useEffect, useState } from 'react';
+import CardList from './CardList';
+import data from '../data.json'; // by remov ing the name of the data.json the ambiguity problem is resove.
```

Comment:
Changes the Json file to dynamic One  and add more data into it if need add custom data.